tests/settings.py

```diff
@@ -0,0 +1,125 @@
+
```

Comment:
What is actually needed in this file?
For example this file is very small:
https://github.com/nansencenter/django-geo-spaas-harvesting/blob/init/geospaas_harvesting/settings.py

Discuss with Adrien and reduce the file content to the absolute required minimum.

Reply:
You didn't remove anything? Why?